packages/wrangler/src/cli.ts

```diff
@@ -1,3 +1,4 @@
+import "dotenv/config"; // Grab locally specified env params from a `.env` file.
```

Comment:
Shouldn't you add `dotenv` to `wrangler/package.json`? It might be working now because of some transient/second-hand dependency.

Reply:
Ah yes! I had added it in my other PR but need to add it here too!
Good spot.

Reply:
Done.